android/app/src/main/AndroidManifest.xml

```diff
@@ -15,6 +15,7 @@
       android:allowBackup="true"
       android:label="@string/app_name"
       android:icon="@mipmap/ic_launcher"
+      android:roundIcon="@mipmap/ic_launcher_round"
```

Comment:
Hey @sworup  - adding this line seems to have changed the android icon

Reply:
@joshxg @Chris-Petty 

It seems android 8 or higher add a rounder launcher icon and adds a background and foreground layer. 

I have two choices here:

1. Remove roundIcons
2. Add png images in resources for round icons

I decided to go with option 2 and add round icons. 

I use Android Studio's image asset generator to generate the icons and xml configuration files. 

Then I replaced the transparent files that were already there in our system with the newly generated while background icons.